NewBank.java

```diff
@@ -6,15 +6,21 @@
 	
 	private static final NewBank bank = new NewBank();
 	private HashMap<String,Customer> customers;
+
+
 	
 	private NewBank() {
 		customers = new HashMap<>();
 		addTestData();
+
 	}
 	
 	private void addTestData() {
 		Customer bhagy = new Customer();
 		bhagy.addAccount(new Account("Main", 1000.0));
+		bhagy.addAccount(new Account("Savings", 500.0));
+		Account a = new Account();
+		a.Transfer(100, "Main", "Savings");
 		customers.put("Bhagy", bhagy);
```

Comment:
I think here you need to actually create account main and savings in order to be able to do a transfer, otherwise the account A has account assigned to it.

Reply:
Hi Lukas,
I thought I had created these in lines 20 and 21 of the NewBank class?
Thanks